### ISSUE TITLE
Call-out expanded: title and fold icon

### ISSUE THREAD

**Issue by DutchPete** [Spekulucius/obsidian-brainhack#9]
I think the screenshots show the problem adequately.

![image](https://github.com/user-attachments/assets/cde46302-3c26-498c-b4ec-bf63e3ca9f93)

![image](https://github.com/user-attachments/assets/ed42910c-e22c-45cc-a2f7-51dfb95f8335)

**Comment by Spekulucius** [comment]
Hey, thanks for the report. I'll look into that, expect a fix by this evening :)

**Comment by Spekulucius** [comment]
I'm sorry, but I wasn't able to reproduce this bug. Can you turn on restricted mode and deactivate all your snippets, if you have any?

My guess would be that it is being caused by your font. Could you change it, and then make another attempt at reproducing this bug?

Which OS are you on?

**Comment by DutchPete** [comment]
![image](https://github.com/user-attachments/assets/70a7ea93-0ce3-425d-8cf1-4007afdef579)

Found the culprit: a snippet, specifically this one:

```
.callout-title-inner {
  font-size: 15px !important;
}
```
Nevertheless, I do find the font size of the call-out title too big. Even when I use Obsidian's default font, i.e. Inter, I still get it:

![image](https://github.com/user-attachments/assets/0db3f566-65bb-47ae-981b-840473d7bb19)

So, how can I reduce the call-out title's font size without getting it crossed through by the dotted line?

OK, I'll make your life easy 😀
If I set the font size to 23px, irrespective of font family, I get this result:

![image](https://github.com/user-attachments/assets/69cf7386-7e96-4251-a050-c955ae568f5a)

It is bigger than what 15px gives, but still smaller than your default. This is acceptable for me. So, we can close this issue and you can concentrate on more important things, OK?

My OS: Linux Mint Debian Edition (LMDE).

**Comment by Spekulucius** [comment]
Ah, this explains the issue. Thank you anyways for the feedback - although changing the callout font size would be too personalized (and special) to include in style settings - because I also thought about adding it there. But seems like you found your solution :)

Thing about sizes: I'd suggest you set sizes (in your snippets for example) not in px, but in em (where 1em = your current font size -> in my theme, callout title font size is 1.4em). That is, because px don't scale with your font size, and if you lower your global font size, your modified callout font size will stay the same.

Personally, I change my font size a lot, because I use it as a "zoom" feature, and set it depending on the distance between me and my screen...

**Comment by DutchPete** [comment]
I set it to 1em but that reduces the font size and brings back the issue, so 1em is not equal to my current 23px.

However, 1 *r*em is **always** (if I am not mistaken) equal to 16px, so my 23px = 1.4375rem. If I use that value in my snippet the font size does not change.

Someone once told me to start using rem (or even em) because px will eventually be dropped. However, I noticed that few people seem to be using those units, the overwhelming majority still uses px, so I reverted to px because I am used to it so the units mean more to me than em/rem. Perhaps I should revert back to using rem; I don't like em because it is not a fixed value, it depends on the "parent", whatever that is.

Another thing is you say that "if you lower your global font size, your modified callout font size will stay the same" — you mean it will stay the same if I use em as units. But if I use px it also stays the same if the global font size is changed, because, as I have always understood it, snippets override everything else — in any case that has been my experience.
